polaris/polaris/sep24/withdraw.py

```diff
@@ -158,7 +159,14 @@ def post_interactive_withdraw(request: Request) -> Response:
             )
             or {}
         )
-        scripts = registered_scripts_func({"form": form, **content})
+        if registered_scripts_func != scripts:
```

Comment:
I haven't tested myself but just wanted to confirm this warning does show up, and also doesn't always show up. Because I see we're assigning [here](https://github.com/stellar/django-polaris/blob/a66d00be0530b680736e168e97b088ef7fa0cc98/polaris/polaris/integrations/javascript.py#L59), and comparing function objects I've found tricky in the past.

Also I think comparing function objects w/ `!=` compares the memory address by default, but maybe better to use `is not` for these, as that always compares addresses. But personal preference here

Reply:
I agree `is not` is better, changing. That should resolve the object comparison issue. I do agree comparing objects is not the greatest way of determining if a custom function was registered. Maybe we should have a better system in place to do that, but thats for a later date.